server/routes/shared_api/autocomplete/helpers.py

```diff
@@ -22,11 +22,41 @@
 import requests
 
 from server.routes.shared_api.autocomplete.types import ScoredPrediction
+from server.routes.shared_api.place import findplacedcid
 
 MAPS_API_URL = "https://maps.googleapis.com/maps/api/place/autocomplete/json?"
 MIN_CHARACTERS_PER_QUERY = 3
 MAX_NUM_OF_QUERIES = 4
 DISPLAYED_RESPONSE_COUNT_LIMIT = 5
+TWO_WORD_CONTINENTS = [{
```

Comment:
Can we just have a single list of additional places instead of one-word and two-word places?

Reply:
The reason I did that is to account the continent hacking properly.
We want to evaluate "Africa" only against one word queries, while "North america" should be evaluated against both one word queries and two word queries in order to account for the queries "North Ame" and "Nort".

If I just have one list of continents, then Id need to do some processing on it to see if there's a space to see if I add it to the two word query too. This seemed a bit cleaner to me, but lmk if you disagree.